conference/talk_voting.py

```diff
@@ -140,6 +141,9 @@ def is_user_allowed_to_vote(user):
 def vote_on_a_talk(request, talk_uuid):
     talk = get_object_or_404(Talk, uuid=talk_uuid)
 
+    if not is_user_allowed_to_vote(request.user):
+        return HttpResponseForbidden('User not allowed to vote')
```

Comment:
Should we give a more detailed reason why this is happening here?

Reply:
Good idea.